data/frameProcessor/src/CalibratorSample.cpp

```diff
@@ -67,7 +71,7 @@ float CalibratorSample::getCMAVal(MemBlockI16& gainBlock, MemBlockF& output, int
     float cmaVal = total / numCMACols;
     if(m_debugRow == row)
     {
-        std::cout << "CMA value for row " << row << ": " << cmaVal << std::endl;
+        LOG4CXX_INFO(m_logger, "CMA value for row " << row << ": " << cmaVal);
```

Comment:
Should this be DEBUG?  It looks like you turn on a debug row, so maybe that's why this is info?

Reply:
no because if you have it on, you want to see it WITHOUT all the normal debug logging.